### ISSUE TITLE
can not work with latest croquet

### ISSUE THREAD

**Issue by wwj718** [NikolaySuslov/aframe-croquet-component#3]
https://www.croquet.io/docs/croquet/

**Comment by NikolaySuslov** [comment]
Thanks for pointing out! The latest Croquet required of using the registered **api_key** (https://croquet.io/keys). Now the component should work.

**Comment by DougReeder** [comment]
The latest code sends the apiKey:
```
        Croquet.Session.join(
            {
                apiKey: '1MAgJydFdvcKpGkHe7bhxLmr3Hj4mofPKvC06mpII',
                appId: "com.aframe.multiuser",           
                name: sessionName,     
                password: password,  
                model: RootModel,
                view: RootView
                //debug: ["session"]
            }

```
but https://unpkg.com/aframe-croquet-component/public/lib/aframe-croquet-component.js maps to v0.0.6, which contains:
```
        Croquet.Session.join(
            {
                appId: "com.aframe.multiuser",           
                name: sessionName,     
                password: password,  
                model: RootModel,
                view: RootView
                //debug: ["session"]
            }

```
So, the inclusion method in the README doesn't work:
`<script src="https://unpkg.com/aframe-croquet-component/public/lib/aframe-croquet-component.js"></script>`

Updating the version number should make that way of inclusion work.

**Comment by NikolaySuslov** [comment]
Thanks! I have updated the version and add apiKey to the Croquet component schema.

**Comment by DougReeder** [comment]
This was resolved by commits a179c8fddc5c86a107b14308dc9947245d386722 and abba53e7204644961953b9d769ed9660308498e8
